walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +272,146 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
-}
 
+  @Test
+  public void optionalWholeDay() {
```

Comment:
make a more readable test name, e.g. for this, maybe queryWithOptionalAttendeeOfWholeDayEvent.

Usually in prod test name can be long. Copied from best practices:

Test names should include the condition being tested and the expected outcome.

A test's name should fully describe what a test does, including not only the state and details of the method being tested but also its expected outcome. For example, a method to transfer money probably contains several interesting behaviors, each of which should be tested individually. Each test should explicitly name not only the method being tested, but also the expected outcome and sometimes specific details of the state or input being tested.
E.g.
transferMoney_deductsFromSource

Reply:
Thanks for the info, done!